tests/actions/notifications/index.test.ts

```diff
@@ -0,0 +1,216 @@
+      const sticky = newSticky()
+      const slidy = newSlidy()
+      slidy.dismiss = { timeout }
+
+      mockStore.dispatch(scheduleNotification(slidy))
+      mockStore.dispatch(scheduleNotification(sticky))
+      expect(mockStore.getActions()).toMatchSnapshot()
+
+      MockDate.set(timeout + 100)
+      jest.runAllTimers()
+      expect(mockStore.getActions()).toMatchSnapshot()
+    })
+  })
+
+  describe('invokeDismiss', () => {
+    beforeEach(() => {
+      mockStore.reset()
+      MockDate.set(0)
+    })
+    afterEach(() => MockDate.reset())
+
+    it('should call the onDismiss callback and remove the notification', () => {
+      const onDismiss = jest.fn()
+      const slidy = newSticky({
+        dismiss: { onDismiss },
+      })
+
+      mockStore.dispatch(scheduleNotification(slidy))
+      expect(mockStore.getActions()).toMatchSnapshot()
+      mockStore.dispatch(invokeDismiss(slidy))
+      jest.runAllTimers()
+      expect(mockStore.getActions()).toMatchSnapshot()
+      expect(onDismiss).toHaveBeenCalledTimes(1)
+    })
+  })
+
+  describe('invokeInteract', () => {
+    beforeEach(() => {
+      mockStore.reset()
+      MockDate.set(0)
+    })
+    afterEach(() => MockDate.reset())
+
+    it('should call the onInteract callback and remove the notification', () => {
+      const onInteract = jest.fn()
+      const sticky = newSticky({
+        interact: {
+          label: 'DO IT',
+          onInteract,
+        },
+      })
+
+      mockStore.dispatch(scheduleNotification(sticky))
+      expect(mockStore.getActions()).toMatchSnapshot()
+      mockStore.dispatch(invokeInteract(sticky))
+      jest.runAllTimers()
+      expect(mockStore.getActions()).toMatchSnapshot()
+      expect(onInteract).toHaveBeenCalledTimes(1)
+    })
+
+    it('should keep the notification if onInteract returns true', () => {
+      const onInteract = jest.fn().mockReturnValue(true)
+      const sticky = newSticky({
+        interact: {
+          label: 'DO IT',
+          onInteract,
+        },
+      })
+
+      mockStore.dispatch(scheduleNotification(sticky))
+      expect(mockStore.getActions()).toMatchSnapshot()
+      mockStore.dispatch(invokeInteract(sticky))
+      expect(mockStore.getActions()).toMatchSnapshot()
+      jest.runAllTimers()
+      expect(onInteract).toHaveBeenCalledTimes(1)
+    })
+  })
+
+  describe('removeNotification', () => {
+    beforeEach(() => {
+      mockStore.reset()
+      MockDate.set(0)
+    })
+    afterEach(() => MockDate.reset())
+
+    it('should remove the notification from the queue', () => {
+      const slidy = newSlidy()
+      const slidy2 = newSlidy()
+
+      mockStore.dispatch(scheduleNotification(slidy))
+      mockStore.dispatch(scheduleNotification(slidy2))
+      mockStore.dispatch(removeNotification(slidy2))
+      expect(mockStore.getActions()).toMatchSnapshot()
+      jest.runAllTimers()
+      expect(mockStore.getActions()).toMatchSnapshot()
```

Comment:
This produces the same snapshot as the one on line `200`, is there a reason for calling `jest.runAllTimers()` on the line above?

Reply:
Runs after `runAllTimers` to ensure things haven't changed